zh-CN/mapping_dev.zh_CN

```diff
@@ -1342,392 +1342,392 @@ dlgSgRenameFile.inf=输入文件的新路径和名称。
-dlgSgRepositoryRemove.chk=Don't show again
-dlgSgRepositoryRemove.fur=This will keep the repositories on disk, but just let SmartGit forget about them.
-dlgSgRepositoryRemove.hdl%1=Are you sure to remove $1 repositories?
-dlgSgRepositoryRemove.tle=Remove
+ 找到仓库 \(将记住选定的仓库\)
+dlgSgRepositoriesSearch.edt"Search In"=搜索中
+dlgSgRepositoriesSearch.hdl=搜索现有本地仓库
+dlgSgRepositoriesSearch.inf=选择应开始搜索的根目录，然后单击 开始搜索 。
+dlgSgRepositoriesSearch.tle=搜索仓库
+dlgSgRepositoryFrameCloseWithoutPush.btn"Close Now"=现在关闭
+dlgSgRepositoryFrameCloseWithoutPush.chk=不再显示
+dlgSgRepositoryFrameCloseWithoutPush.fur=你有可推送的提交。也许你想在关闭这个窗口之前推它们。
+dlgSgRepositoryFrameCloseWithoutPush.hdl=你确定在没有推送提交的情况下关闭吗？
+dlgSgRepositoryFrameCloseWithoutPush.tle=关闭
+dlgSgRepositoryRemove.btn"Remove"=删除
+dlgSgRepositoryRemove.chk=不再显示
+dlgSgRepositoryRemove.fur=这会将仓库保留在磁盘上，但只是让 SmartGit 忘记它们。
+dlgSgRepositoryRemove.hdl%1=您确定要删除 $1 的仓库吗？
+dlgSgRepositoryRemove.tle=删除
 dlgSgRepositorySettings.chk"Always fetch new commits, tags and branches from submodule"=\
- Always fetch new commits, tags and branches from submodule
-dlgSgRepositorySettings.chk"Initialize new submodules"=Initialize new submodules
-dlgSgRepositorySettings.chk"Prune obsolete remote tracked branches"=Prune obsolete remote tracked branches
+ 始终从子模块中获取新的提交，标记和分支
+dlgSgRepositorySettings.chk"Initialize new submodules"=初始化新的子模块
+dlgSgRepositorySettings.chk"Prune obsolete remote tracked branches"=删除过时的远程跟踪分支
 dlgSgRepositorySettings.chk"Remember as default"=记住默认
 dlgSgRepositorySettings.chk"Sign all commits"=签名所有提交
 dlgSgRepositorySettings.chk"Update registered submodules"=更新已注册的子模块
 dlgSgRepositorySettings.edt"Email"=电子邮箱
-dlgSgRepositorySettings.edt"GPG Program"=GPG Program
+dlgSgRepositorySettings.edt"GPG Program"=GPG 程序
 dlgSgRepositorySettings.edt"Name"=用户名
-dlgSgRepositorySettings.edt"Signing Key"=Signing Key
-dlgSgRepositorySettings.edt"Submodule handling\*"=Submodule handling\*
+dlgSgRepositorySettings.edt"Signing Key"=签名密钥
+dlgSgRepositorySettings.edt"Submodule handling\*"=子模块处理\*
 dlgSgRepositorySettings.edt"Text File Encoding"=文本编码
-dlgSgRepositorySettings.hdl=编辑有效的版本库设置
-dlgSgRepositorySettings.inf=在这里，您可以查看和编辑用户配置文件和版本库的有效Git设置。要将选项保存为用户设置，请选择“记住默认”。
-dlgSgRepositorySettings.tle=版本库设置
+dlgSgRepositorySettings.hdl=编辑有效的仓库设置
+dlgSgRepositorySettings.inf=在这里，您可以查看和编辑用户配置文件和仓库的有效 Git 设置。要将选项保存为用户设置，请选择 “记住默认” 。
+dlgSgRepositorySettings.tle=仓库设置
 dlgSgResetAdv.btn"Reset"=重置
 dlgSgResetAdv.chk"Thoroughly fix line endings according to .gitattributes"=\
- Thoroughly fix line endings according to .gitattributes
-dlgSgResetAdv.hdl%1=Reset to commit $1
-dlgSgResetAdv.inf=Reset the current branch head to the selected commit and optionally update Index and working tree.
+ 根据 .gitattributes 彻底修复行尾
+dlgSgResetAdv.hdl%1=重置提交 $1
+dlgSgResetAdv.inf=将当前分支头重置为所选提交，并可选择更新索引和工作树。
 dlgSgResetAdv.rbt"Don't touch the Index nor the working tree - 'soft'"=\
- Don't touch the Index nor the working tree - 'soft'
+ 不要触摸索引或工作树 - “软”
```

Comment:
Translation here as "soft" seems to be somewhat inappropriate. @syntevo What mean about this 'Soft' and next 'hard'.

Reply:
@mstrap What mean about this 'Soft' and next 'Hard'.

Reply:
`--soft` and `--hard` are exactly the command line parameters for `git --reset`.